templates/parachain/README.docify.md

```diff
@@ -240,6 +241,127 @@ npx @acala-network/chopsticks@latest --chain-spec <path/to/chain_spec.json>
 relay chain network (see [Parachain Template node](#parachain-template-node) setup).
 
+## Deploy to Paseo TestNet And See Your First Block
+
+After setting up your parachain locally, you can deploy it to the Paseo public testnet to observe it producing blocks in a realistic environment.
+
+### Steps to Deploy
+
+Note: The following instructions are also available at the [Zero to Hero Tutorial](https://docs.polkadot.com/tutorials/polkadot-sdk/parachains/zero-to-hero/deploy-to-testnet/) where this is available in a more detailed manner, we recommend you to try that out as well if something here is unclear.
+
+### Reserve & Register a ParaID
+
+1. Open [Polkadot.js Apps](https://polkadot.js.org/apps/#/explorer) and switch the network selector to Paseo.
+
+2. Go to Network → Parachains → Parathreads, click `+ ParaId`, then Submit. You’ll be assigned the next free ID (e.g. 4508).
+
+3. In Explorer, confirm you see a `registrar.Reserved` event.
+
+### Register Your Parachain
+
+1. Ensure your parachain's genesis state and runtime .wasm file are ready.
+
+2. In Polkadot.js Apps, navigate to Network > Parachains, then to the Parathreads tab.
+
+3. Click on `+ Register` and provide:
+  1. Your reserved ParaID
+  2. The compiled .wasm runtime file
+  3. The genesis state file
+
+4. Submit the transaction to register your parachain.
+
+### Obtain PAS Tokens
+
+Visit the [Polkadot Faucet](https://faucet.polkadot.network/) and request PAS tokens for your account. Ensure you're connected to the Paseo network on [Polkadot.js Apps](https://polkadot.js.org/apps/#/explorer).
+
+### Acquire Coretime
+
+To enable your parachain to produce and finalize blocks, you need to obtain coretime.
+
+1. In Polkadot.js Apps, go to Developer > Extrinsics.
+
+2. Select your account and choose the onDemand.placeOrderAllowDeath extrinsic.
+
+3. Provide the following parameters:
+
+  1. paraId: Your reserved ParaID
+  2. maxAmount: An appropriate amount of PAS tokens (e.g., 1000000000000)
+
+4. Submit the extrinsic.
+
+Upon success, your parachain will start producing blocks.
+
+### Generate Customs Keys for Collator Node
+
+1. To perform this step, you can use subkey, a command-line tool for generating and managing keys:
+```
+docker run -it parity/subkey:latest generate --scheme sr25519
+```
+
+### Build the Chain Specification
+Generate the plain chain specification:
+```
+polkadot-omni-node chain-spec-builder \
+  --chain-spec-path ./plain_chain_spec.json \
+  create \
+  --relay-chain paseo \
+  --para-id <YOUR_PARA_ID> \
+  --runtime target/release/wbuild/parachain-template-runtime/parachain_template_runtime.compact.compressed.wasm \
+  named-preset local_testnet 
+```
+Edit plain_chain_spec.json to update fields like name, id, protocolId, para_id, and parachainInfo.parachainId with your ParaID. Also, configure balances, collatorSelection.invulnerables, session.keys, and sudo fields accordingly.
+Then, generate the raw chain specification:
```

Comment:
Right, but I think this should've been done sometime before already, when generating the genesis-state?